packages/zoe/test/unitTests/contracts/fakeVatAdmin.js

```diff
@@ -7,4 +7,7 @@ export default harden({
       root: E(evalContractBundle(bundle)).buildRootObject(),
     });
   },
+  createVatByName: _name => {
+    throw Error(`createVatByName not supported in fake mode`);
```

Comment:
Ah, is it only for testing that we don't use `createVatByName`? Could we add a comment to that effect around line 243 of zoe.js?

Reply:
A lot of the unit tests (maybe all of them, but I haven't surveyed comprehensively) don't even use SwingSet; `fakeVatAdmin` is some slight-of-hand to enable that to work.  Since there's no swingset, there's no kernel bundle table to look up bundles in either, so `createByName` is not an option in that case.   On the other hand, since these cases aren't using SwingSet they're not shipping bundles over the wire either, so I think it's OK.  But Zoe doesn't know which of these cases it's in.  The code around line 243 of zoe.js is partially in support of this case, but not just that since I was also concerned about backwards compatibility.

Reply:
Yeah, anything labeled unitTest rather than SwingsetTest doesn't use Swingset at all. 

Ok, that makes sense. I think backwards compatibility isn't a big advantage now since our external usage is minimal and we control our internal usage. It would be nice to to be able to look at the Zoe code and know from a comment exactly why there are two options and when we can get rid of one of them. If that's not possible, that's ok though.

Reply:
A comment will do no harm though.  I will update.